sw/host/opentitanlib/src/transport/hyperdebug/c2d2.rs

```diff
@@ -0,0 +1,74 @@
+use anyhow::{bail, Result};
```

Comment:
This file needs a license header or it won't pass CI.

Reply:
Done